### DISCUSSION TITLE
Exported Types Wrong?

### DISCUSSION THREAD

**Discussion by joewood** [csandman/chakra-react-select#106] [Q&A] [answered]
Thanks for creating this component. However, the types shipped with the component reuse the Chakra props of Select, rather than the different props that react-select use (for example, `isMulti` is not supported and you can't set the `options`).

**Comment by HasseNasse** [comment]
Facing the same issue..

**Comment by csandman** [comment]
For the most part I'm trying to expand the types offered by react-select, check these files:

- https://github.com/csandman/chakra-react-select/blob/main/src/chakra-react-select.tsx
- https://github.com/csandman/chakra-react-select/blob/main/src/types.ts

However I'm unfortunately very new to typescript so there's a fair chance I didn't export these right.  If you have any recommendations to improve it, please let me know

**Comment by Patrascu-Lucian** [comment]
Working fine for me: https://codesandbox.io/s/chakra-react-select-ts-test-z131f?file=/src/App.tsx

**Comment by joewood** [comment, marked as answer]
You're right @Patrascu-Lucian - the issue seemed to be related to a type conflict between react-select and chakra-react-select. Upgrading everything to latest seems to have resolved it. Thanks all.
